src/views/Settings.vue

```diff
@@ -33,7 +33,62 @@
         @input="setMinLowerChar"
       />
     </div>
+  <div class = "configurations">
+    <h1>Configurations</h1>
+      <ul>
+        <li><button class = "button1" v-on:click="
```

Comment:
`
```suggestion
        <li><button class = "button1" @click="
```
Can optionally use `@click`, see [Vue v-on shorthand](https://vuejs.org/v2/guide/syntax.html#v-on-Shorthand)